pypeit/scripts/view_fits.py

```diff
@@ -33,11 +33,14 @@ def get_parser(cls, width=None):
                             help='Show a FITS extension in the raw file. Note --proc and --mosaic '
                                  'will not work with this option.')
         parser.add_argument('--det', type=str, default='1', nargs='*',
-                            help='Detector(s) to show.  If more than one, the list of detectors '
-                                 'must be one of the allowed mosaics hard-coded for the selected '
+                            help='Detector(s) to show.  If more than one, the list of detectors, i.e. --det 4 8 '
```

Comment:
is it `4 8` or `4,8` or are both allowed?

Reply:
Good question.  It actually looks like it has to be '4 8', and '4,8' will likely barf.  We should think about changing this.

Reply:
Yeah, I think MSC01 should also be allowed, which it is not.